src/config.rs

```diff
@@ -199,6 +199,19 @@ fn recursive_extend_map(
     }
 }
 
+/// Merge two TOML tables
```

Comment:
This function already exists: 
https://github.com/SuperCuber/dotter/blob/1712b25ff8f833aa29afd9b4cfd39f1ef3eed2dd/src/config.rs#L179

You should either rewrite the below code to use it or rewrite the code that uses that function to use this instead

Reply:
Oh wow, I'm sorry, how did I not notice that? Will do, thank you!